Package.swift

```diff
@@ -1,13 +1,13 @@
 // swift-tools-version:5.4
 import PackageDescription
 
-let checksum = "c918e10d98034153484f38b5f6529eb7b062e40e97be27e8f79944dfe58aef82"
-let version = "114.20230504050432"
-let url = "https://firefox-ci-tc.services.mozilla.com/api/index/v1/task/project.application-services.v2.swift.114.20230504050432/artifacts/public%2Fbuild%2FMozillaRustComponents.xcframework.zip"
+let checksum = "8d9c4ce1f50b94bf953828d40e87c693ca0b1108805b3373f3d037fbe6e12b98"
+let version = "114.0"
```

Comment:
heh nit to fix on main after this, the `url` and `focusUrl` don't reference the `version` here